test/it/uk/gov/hmrc/exports/connector/CustomsDataStoreConnectorSpec.scala

```diff
@@ -49,6 +48,40 @@ class CustomsDataStoreConnectorSpec extends IntegrationTestSpec {
       }
     }
 
+    "email service responds with OK (200) and deliverable = false flag" should {
```

Comment:
"deliverable = false" isn't really accurate. Email service actually responds with a payload that contains the `undeliverable` json field.